lib/ex_doc/language/erlang.ex

```diff
@@ -303,7 +303,13 @@ defmodule ExDoc.Language.Erlang do
         inner
 
       _ ->
-        ast
+        case ExDoc.Language.Elixir.custom_link(attrs, config) do
+          nil ->
```

Comment:
I also understand this is rather ugly, but maybe having the function moved can help me understand better how to code this bit...